api/models/graphite_render_meta.go

```diff
@@ -23,12 +23,18 @@ func (rwm ResponseWithMeta) MarshalJSONFast(b []byte) ([]byte, error) {
 // RenderMeta holds metadata about a render request/response
 type RenderMeta struct {
 	Stats stats
+	StorageStats
```

Comment:
This just seems hacky.  Lets clean it up and use `Stats` and `StorageStats`  consistently.   eg
```
type RenderMeta struct {
	RenderStats
	StorageStats
}
```